Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/Filters/RelationshipsAuthorizationFilters.cs

```diff
@@ -33,381 +35,232 @@ public static class RelationshipsAuthorizationFilters
         private static readonly Lazy<FilterApplicationDetails> _educationServiceCenterIdToEducationServiceCenterId =
             new Lazy<FilterApplicationDetails>(() => CreateClaimValuePropertyFilter("EducationServiceCenterId"));
 
-        private static readonly Lazy<FilterApplicationDetails> _educationServiceCenterIdToStudentUSI
+        private static readonly Lazy<FilterApplicationDetails> _educationServiceCenterIdToStudentUSI =
+            new Lazy<FilterApplicationDetails>(() => new ViewFilterApplicationDetails(
+                "EducationServiceCenterIdToStudentUSI",
+                "EducationOrganizationIdToStudentUSI",
+                "SourceEducationOrganizationId",
+                "StudentUSI"));
+
+        private static readonly Lazy<FilterApplicationDetails> _educationServiceCenterIdToStudentUSIThroughResponsibility
             = new Lazy<FilterApplicationDetails>(
                 () =>
-                    new FilterApplicationDetails(
-                        "EducationServiceCenterIdToStudentUSI",
-                        @"StudentUSI IN (
-                            SELECT {newAlias1}.StudentUSI
-                            FROM auth.EducationOrganizationIdToStudentUSI {newAlias1}
-                            WHERE {newAlias1}.SourceEducationOrganizationId IN (:SourceEducationOrganizationId))",
-                        @"{currentAlias}.StudentUSI IN (
-                            SELECT {newAlias1}.StudentUSI
-                            FROM " + "auth_EducationOrganizationIdToStudentUSI".GetFullNameForView() + @" {newAlias1}
-                            WHERE {newAlias1}.SourceEducationOrganizationId IN (:EducationServiceCenterId))",
-                        (c, w, p, jt) => c.ApplyJoinFilter(
-                            w, p, "EducationOrganizationIdToStudentUSI", "StudentUSI", "StudentUSI", "SourceEducationOrganizationId",
-                            jt, Guid.NewGuid().ToString("N")
-                            ),
-                        (t, p) => p.HasPropertyNamed("StudentUSI")));
+                    new ViewFilterApplicationDetails(
```

Comment:
I really like the extractions done here. It makes this class a bit more readable.